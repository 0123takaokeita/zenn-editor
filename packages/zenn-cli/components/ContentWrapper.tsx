import React from 'react';

type Props = { children: React.ReactNode };
export const ContentWrapper: React.FC<Props> = ({ children }) => {
  return <div className="content-wrapper">{children}</div>;
};
