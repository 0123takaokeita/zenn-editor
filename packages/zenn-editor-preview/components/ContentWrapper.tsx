const ContentBody: React.FC<{ children: React.ReactNode }> = ({ children }) => {
  return <div className="content-wrapper">{children}</div>;
};
export default ContentBody;
