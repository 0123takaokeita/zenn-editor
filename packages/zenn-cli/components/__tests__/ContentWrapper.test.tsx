import React from 'react';
import { createRenderer } from 'react-test-renderer/shallow';
import { ContentWrapper } from '@components/ContentWrapper';

const renderer = createRenderer();

describe('<ContentWrapper/>', () => {
  it('should render and match the snapshot', () => {
    renderer.render(
      <ContentWrapper>
        <div />
      </ContentWrapper>
    );
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
