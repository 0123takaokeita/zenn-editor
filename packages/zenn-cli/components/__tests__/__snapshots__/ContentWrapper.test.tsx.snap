// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentWrapper/> should render and match the snapshot 1`] = `
<div
  className="content-wrapper"
>
  <div />
</div>
`;
